docs/make_docs_without_docker.sh

```diff
@@ -29,5 +29,5 @@ python sphinx-build.py -Q -b html autogenerated_source build
 python correct_built_html_files.py
 
 # delete the code
-./remove_files.sh
+bash ./remove_files.sh
```

Comment:
are you sure the `./` is needed with the prepending `bash` now in place?

Reply:
Worked on my system either way. Still have removed the `./`